packages/web3-eth-contract/test/integration/contract_methods.test.ts

```diff
@@ -148,23 +148,35 @@ describe('contract', () => {
 			});
 
 			it('should returns errors on reverts', async () => {
-				await expect(
-					contractDeployed.methods.reverts().send(sendOptions),
-				).rejects.toMatchObject({
-					name: 'TransactionRevertedWithoutReasonError',
-					receipt: {
-						cumulativeGasUsed: BigInt(21543),
-						from: acc.address,
-						gasUsed: BigInt(21543),
-						logs: [],
-						logsBloom:
-							'0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
-						status: BigInt(0),
-						to: contractDeployed.options.address?.toLowerCase(),
-						transactionIndex: BigInt(0),
-						type: BigInt(2),
-					},
-				});
+				// hardhat does not support eip-838
```

Comment:
I do not think hardhat does not support eip-838. Are you sure? Or should we create a follow up issue to investigate more into this?

Reply:
You are correct, I created an issue. thank you 
https://github.com/web3/web3.js/issues/6772